feature/step_definitions/stepdefs.js

```diff
@@ -0,0 +1,2 @@
+const assert = require("assert");
+const { Given, When, Then } = require('cucumber');
```

Comment:
Remember the extra newline at the end of the file.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
good point. Fixed